tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/GivenUsingSuite.scala

```diff
@@ -339,44 +330,14 @@ class GivenUsingSuite extends BaseDottySuite {
   }
 
   test("given-alias-inline-subtype") {
-    runTestAssert[Stat]("inline given intOrd: ? <: Ord[Int] = ???")(
```

Comment:
This doesn't parse in Scala 3 so I changed the test case.

Reply:
:+1: (change a test name to not include 'subtype')

Reply:
Fixed!